devices.js

```diff
@@ -5746,6 +5762,24 @@ const devices = [
             execute(device, actions, callback);
         },
     },
+
+    // Lonsonho
+    {
+        zigbeeModel: ['Plug_01'],
+        model: 'Zigbee plug',
+        vendor: 'Lonsonho',
```

Comment:
https://nl.aliexpress.com/item/32991703204.html is this the device?

Reply:
no. this https://ru.aliexpress.com/item/4000116784070.html